packages/react/src/components/Menu/Menu.tsx

```diff
@@ -225,5 +222,9 @@ class Menu extends AutoControlledComponent<ReactProps<MenuProps>, MenuState> {
 }
 
 Menu.create = createShorthandFactory({ Component: Menu, mappedArrayProp: 'items' })
+Menu.slotClassNames = {
```

Comment:
lets follow the same approach that is introduced for other components (i.e. use the dedicated `static` field for now)